creator-node/src/routes/files.js

```diff
@@ -196,7 +186,7 @@ const getDirCID = async (req, res) => {
     // Cat 1 byte of CID in ipfs to determine if file exists
     // If the request takes under 500ms, stream the file from ipfs
     // else if the request takes over 500ms, throw an error
-    await ipfsSingleByteCat(ipfsPath, req, 500)
+    await ipfsSingleByteCat(ipfsPath, req.logContext, 500)
```

Comment:
same question here, do we really need to do this any longer?

Reply:
I think we should keep it :) 
f8dccdc0e44edd5fa2a21049c682dceea5d2d963